tester_app/client/app/bundles/HelloWorld/startup/helloWorldAppClient.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import createStore from '../store/helloWorldStore';
+import HelloWorld from '../containers/HelloWorld';
+
+const helloWorldAppClient = props => {
```

Comment:
@justin808 okay I will have to incorporate this change again as it's not in the latest batch as of the time I am writing this (v28). But what of the fact that we are using a generator function but passing `generator_function: false` in the view and yet it is still working perfectly fine?